src/simtools/applications/run_application.py

```diff
@@ -137,9 +137,7 @@ def read_application_configuration(configuration_file, steps, logger):
     application_config = gen.collect_data_from_file(configuration_file).get("CTA_SIMPIPE")
     place_holder = "__SETTING_WORKFLOW__"
     workflow_dir, setting_workflow = get_subdirectory_name(configuration_file)
-    output_path = (
-        workflow_dir.with_name(workflow_dir.name.replace("input", "output")) / setting_workflow
-    )
+    output_path = Path(str(workflow_dir).replace("input", "output") + "/" + setting_workflow)
```

Comment:
We went back to string manipulations? =)

Reply:
The previous one should also work, you just need to cast to path as in

`Path(workflow_dir).with_name(workflow_dir.name.replace("input", "output")) / setting_workflow`

One shouldn't do string manipulations for paths...

Reply:
Fixed it - I think our issue was that we did not expect that `Path.parts` returns strings (obviously...these are parts of a Path).